### DISCUSSION TITLE
특정 이벤트 후 상점 오픈 기능

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#231] [5. Q&A]
안녕하세요.
제페토 공식 맵 중에 상점을 열어서 아바타를 커스터마이징 하는 기능이 있는데,
유니티로 만든 제페토 월드에서 특정 이벤트 이후에 상점을 여는 것이 가능할까요?
아래와 같은 process를 생각하고 있습니다.

1. 화면 오브젝트 터치로 일정 시간 동안 오브젝트가 움직이는 애니메이션 재생
2. 애니메이션 종료 후 상점 오픈과 동시에 아이템 적용
 
가이드 부탁드립니다.
감사합니다.

![image](https://user-images.githubusercontent.com/5335151/150280329-654dd4a4-0a36-4ad2-86cf-1eafa01cef10.png)
![image](https://user-images.githubusercontent.com/5335151/150280350-e07c06d3-30c5-4c02-b736-f677331186d4.png)
![image](https://user-images.githubusercontent.com/5335151/150280369-aeb6a0db-f77b-4b67-8a98-9c0f8ca7bdb0.png)

https://user-images.githubusercontent.com/5335151/150279922-35673458-3afa-41e1-bad3-1e9e13d058b9.mp4

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재는 제페토의 상점과 오브젝트를 연동하는 기능은 제공하고 있지 않습니다.

감사합니다.

**Reply by DoLongJu** [reply]
혹시 추후에 업데이트 예정인 기능인가요?
